articles/azure-monitor/app/asp-net-core.md

```diff
@@ -110,6 +110,14 @@ The [Application Insights SDK for ASP.NET Core](https://nuget.org/packages/Micro
     > [!NOTE]
     > An instrumentation key specified in code wins over the environment variable `APPINSIGHTS_INSTRUMENTATIONKEY`, which wins over other options.
 
+### User secrets and other configuration providers
+
+In case you want to store the isntrumentation key in ASP.NET Core User Secrets or retrieve it from another configuration provider, you can utilize the overload with a `Microsoft.Extensions.Configuration.IConfiguration` parameter:
```

Comment:
```suggestion
In case you want to store the instrumentation key in ASP.NET Core User Secrets or retrieve it from another configuration provider, you can utilize the overload with a `Microsoft.Extensions.Configuration.IConfiguration` parameter:
```